content/logs/aws.md

```diff
@@ -8,50 +8,81 @@ beta: true
     {{< img src="logs/aws/basic_settings.png" alt="Basic Settings" responsive="true" >}}
-6. Scroll back to the top of the page and hit **Save and test**
+7. Scroll back to the top of the page and hit **Save.**
 
 ### Test your Lambda
 
-1. Configure the test event. Select **Cloudwatch Logs** as the sample event.
+1. Press **Test**. 
+2. Search for and select **Cloudwatch Logs** as the sample event.
     {{< img src="logs/aws/test_event.png" alt="Test Event" responsive="true" >}}
-2. Save and test.
+2. Give the event a unique name and press **Create**.
+3. Press Test and ensure the test passes with no errors.
 
 ## Collection
 
-Your lambda function is now ready to send logs to your Datadog platform.
-Setup the relevant triggers for each AWS service you want to monitor.
-
-### S3, Cloudwatch, API Gateway, Kinesis and SNS
-In your lambda, go in the triggers tab and select `Add Trigger`:
+Your Lambda function is now ready to send logs to the Datadog platform. Setup the relevant triggers for each AWS service you want to monitor.
+
+**The Lambda function you just created must be in the same region as the S3 bucket you are using as a trigger. If you have logs across multiple regions you must create additional Lambda funcions.**
+
+### Automatically set up triggers
+If you are storing logs in many S3 buckets, Datadog can automatically manage triggers for you.
+
+1. Add the required permissions to your Datadog role in the [IAM Console](https://console.aws.amazon.com/iam/home#/roles). You may already have some of these permissions from our other AWS integrations. Information on how these permissions are used can be found in the [permissions](#permissions) section below:
+
+{{< highlight json>}}
+"elasticloadbalancing:DescribeLoadBalancers",
+"elasticloadbalancing:DescribeLoadBalancerAttributes",
+"lambda:AddPermission",
+"lambda:GetPolicy",
+"lambda:RemovePermission",
+"s3:GetBucketLogging",
+"s3:GetBucketLocation",
+"s3:GetBucketNotification",
+"s3:ListAllMyBuckets",
+"s3:PutBucketNotification"
+{{< /highlight >}}
+
+2. Navigate to the *Collect Logs* tab in the [AWS Integration tile](https://app.datadoghq.com/account/settings#integrations/amazon_web_services)
+3. Select the AWS Account from where you want to collect logs, and enter the ARN of the Lambda created in the previous section.
+{{< img src="logs/aws/AWSLogStep1.png" alt="Enter Lambda">}}
+4. Check off the services from which you'd like to collect logs and hit save. To stop collecting logs from a particular service, simply uncheck it.
+{{< img src="logs/aws/AWSLogStep2.png" alt="Select services">}}
+5. If you have logs across multiple regions, you must create additional Lambda functions in those regions and enter them in this tile.
+6. To stop collecting all AWS logs, press the *x* next to each Lamdba ARN. All triggers for that function will be removed. 
+7. Within a few minutes you should see your AWS Logs appear in our [logging platform](https://app.datadoghq.com/logs).
```

Comment:
The `few minutes` might be confusing here. 

It can indeed take a couples minutes for the first logs to show up in the platform but then it should take only a couples seconds. 
Maybe we can try to be clearer to avoid users to think there is a delay for all logs as we have for Cloudwatch metrics (for instance).